FFXIV_TexTools/Views/ModPack/Wizard/WizardModPackControl.xaml.cs

```diff
@@ -225,23 +226,30 @@ private void OptionsList_SelectionChanged(object sender, SelectionChangedEventAr
             {
                 OptionDescriptionTextBox.Text = option.Description ?? string.Empty;
 
-                if (option.Image != null && !option.Image.IsDisposed)
+                if (option.Image != null)
                 {
-                    BitmapImage bmp;
+                    try
+                    {
+                        BitmapImage bmp;
+
+                        using (var ms = new MemoryStream())
+                        {
+                            option.Image.Save(ms, new BmpEncoder());
+
+                            bmp = new BitmapImage();
+                            bmp.BeginInit();
+                            bmp.StreamSource = ms;
+                            bmp.CacheOption = BitmapCacheOption.OnLoad;
+                            bmp.EndInit();
+                            bmp.Freeze();
+                        }
 
-                    using (var ms = new MemoryStream())
+                        OptionPreviewImage.Source = bmp;
+                    }
+                    catch(ObjectDisposedException)
```

Comment:
`Image.IsDisposed` was made private in one of the updates to ImageSharp and instead they opted for throwing a `ObjectDisposedException` as can be seen in their source below. I think this is equivalent to what we had before.

https://github.com/SixLabors/ImageSharp/blob/master/src/ImageSharp/Image.cs#L161